python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_node_type.schema.yml

```
@@ -97,6 +97,18 @@ $defs:
               enabled:
                 type: bool
                 default: false
+              downlinks:
+                type: dict
+                keys:
+                  enabled:
+                    type: bool
+                    description: Add all underlay downlinks of the switch as the downstream interfaces of the configured Link Tracking Group(s).
```

As they will only be the downstreams of the one link-tracking group only

```suggestion
                    description: Add all underlay downlinks of the switch as the downstream interfaces of the configured Link Tracking Group.
```

I think it could also be helpful if we explain here how specific link-tracking group will be selected (explicitly specified -> first -> default)

---

UPdated the description